RELEASES.md

```diff
@@ -4,7 +4,7 @@
 
 ## 0.25.0
 
-Not yet released
+Released 2020-03-15.
```

Comment:
```suggestion
Released 2021-03-15.
```

Reply:
(we're the 16th btw, not sure if today's date was what you were going for ?)

Reply:
Double d'oh.